src/renderer/shared/api/storage/common/upgrades.ts

```diff
@@ -27,3 +33,31 @@ export const upgradeEvents = async (trans: Transaction) => {
     trans.table('multisigEvents').bulkAdd(newEvents),
   ]);
 };
+
+// /**
```

Comment:
Do you need this commented code? 

`xcmDestination` will be just `undefined`, maybe it's enough? Or you think when you add index for this field it will be failed?